tensorflow_serving/model_servers/model_management_impl.cc

```diff
@@ -0,0 +1,16 @@
+#include "tensorflow_serving/model_servers/model_management_impl.h"
+#include "tensorflow_serving/util/status_util.h"
+
+namespace tensorflow {
+    namespace serving {
+        Status ModelManagementImpl::Load(ServerCore *core,
+                                              const LoadModelConfigurationRequest* request,
+                                              LoadModelConfigurationResponse *response) {
+
+            const Status &status = core->ReloadConfig(request->configuration());
```

Comment:
nit: drop the & in front off 'status' and 'statusProto'

Reply:
CLion autocompletion... Is it possible to create IDE-specific dir with project codestyle configs? Would save time for future reviews if most of the code would be using it automatically